ember/app/ui/components/work-item-list/item/component.js

```diff
@@ -85,4 +67,25 @@ export default class WorkItemListItemComponent extends Component {
       this.args.workItem.id
     );
   }
+
+  @dropTask
+  *getIdentity() {
+    let idpId = null;
+
+    if (
+      !this.args.workItem.assignedUser &&
+      this.args.workItem.raw.assignedUsers[0]
```

Comment:
Mix between `workItem` and `workItem.raw` intended?

Reply:
For assignedUsers it shouldn't be, but for closedByUser it should use the raw, as a function overwrites the value
Changed that